File: Lógica/Lista2/ExPySala/ex2.py
nota1 = float(input("Digite a nota 1: "))
nota2 = float(input("Digite a nota 2: "))

media = (nota1 + nota2)/2

if media >= 6:
    print(f"Aprovado. Média: {media:.2f}")
else:
    nota_auxiliar = float(input("Reprovado. Digite a nota auxiliar: "))
    media = (nota1 + nota2 + nota_auxiliar) / 3
    if media >= 5:
        print(f"Aprovado. Média: {media:.2f}")
    else:
        print(f"Reprovado. Média: {media:.2f}")

    

File: Lógica/Lista2/ExPySalaC/ex2.c
#include <stdio.h>

void main()
{   
    float nota1,nota2,media,nota_auxiliar;
    
    printf("Digite a nota 1: ");
    scanf("%f",&nota1);
    
    printf("Digite a nota 2: ");
    scanf("%f",&nota2);

    media = (nota1 + nota2) / 2;
    
    if (media >= 6){
        printf("Aprovado. Média: %.2f",media);
    }
    else{
        printf("Reprovado. Digite a nota auxiliar: ");
        scanf("%f",&nota_auxiliar);
        media = (nota1 + nota2 + nota_auxiliar) / 3;
        
        if (media >=5){
            printf("Aprovado. Média: %.2f", media);
        }
        
        else{
            printf("Reprovado. Média: %.2f", media);
        }
    }
}